libnd4j/include/graph/RandomGenerator.h

```diff
@@ -139,8 +139,11 @@ namespace nd4j {
 
         FORCEINLINE RandomGenerator::RandomGenerator(Nd4jLong rootSeed, Nd4jLong nodeSeed) {
             // this seed is used graph-level state
-            if (rootSeed == 0)
-                rootSeed = currentMilliseconds();
+            if (rootSeed == 0){
+                auto s = std::chrono::system_clock::now().time_since_epoch();
+                rootSeed  = std::chrono::duration_cast<std::chrono::milliseconds>(s).count();
```

Comment:
This is basically what currentMilliseconds() does, how is this different?

Reply:
In cases when inlining is deactivated (happened with ncc at one point), this results in a constructor calling a member function, which is generally fine, except in one case: https://stackoverflow.com/questions/3091833/calling-member-functions-from-a-constructor/31757790#31757790

I can try to retrieve the case where this was happening, one of the tests was failing because of that.

Reply:
We can make currentMilliseconds() static, that should be fine, right?

Reply:
Probably. The constructor used to call SetStates, which was problematic, but currentMilliseconds() is probably no problem.

Reply:
Ok, let's try to make it static, and revert these lines to call that function again.